scripts/README.md

```diff
@@ -18,8 +44,9 @@ Follow the instructions below to load the environments variables required for th
    ```
    **Note**: you may also want to add it to your shell's startup file so you don't need to run it for every terminal session.
 4. Run your script
+</details>
 
-## Available Scripts
+## Available_Scripts
```

Comment:
Was the underscore added by mistake?

Reply:
I added it because the README was having issues finding the `Available Scripts` reference in the ToC. I'll play around with it more to see if I can get it working with a space

Reply:
I was able to get the generate to work with spacing